server/routes/server.js

```diff
@@ -11,8 +11,20 @@ function convertToClientFormat(selected_config, esResponse) {
       var flatten = require('flat');
       source = flatten(source);
     }
+    
+    var display = source[selected_config.fields.mapping['display_timestamp']];
+    
+    if (selected_config.format_timestamp != null){
+      var moment = require('moment-timezone');
+      display = moment(source[selected_config.fields.mapping['timestamp']]);
```

Comment:
That's not a bad idea, but I'm not sure how to use the same config setting to hold either a format string or a field name. My thought was that specifying format_timestamp would indicate that there was not a field that had the data formatted as the user desired, and instead wanted to specify a format for the output that meets their needs. For that reason it supersedes the display_timestamp setting.

For example, I currently map @timestamp to both timestamp and display_timestamp, but needed a way to clean it up since I don't like the standard way that it (as a datetime field) is printed by ES.